charts/azure-k8s-metrics-adapter/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for azure-k8s-metrics-adapter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: jsturtevant/azure-k8s-metrics-adapter-amd64
+  tag: v0.3.8-90
+  pullPolicy: Always
+
+logLevel: 2
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+adapterSecurePort: 6443
+
+apiServiceInsecureSkipTLSVerify: true
+apiServiceGroupPriorityMinimum: 100
+apiServiceVersionPriority: 100
+
+service:
+  type: ClusterIP
+  port: 443
+
+# Azure Configuration
+
+azureAuthentication:
+  # method: {msi,clientSecret,clientCertificate}
+  method: msi
+  tenantID: ""
+  clientID: ""
+  clientSecret: ""
+  clientCertificate: ""
+  clientCertificatePath: ""
+  clientCertificatePassword: ""
+
+# Override the subscription if outside of Azure or for full control with `SUBSCRIPTION_ID`
+# See https://github.com/jsturtevant/azure-k8-metrics-adapter#subscription-information
+defaultSubscriptionId: ""
+
+extraEnv: {}
+extraArgs: {}
+
+# This is used by the static manifest generator in order to create a static
+# namespace manifest for the namespace that cert-manager is being installed
```

Comment:
what is cert-manager?  Not sure I know what this `createNamedspaceResource` does?  Can't we use  `helm install --namespace`  to install to a name space.  In fact I think we should add `--namespace` to the readme to direct users to install into a namespace called "custom Metrics". this way users get off the right start.

Reply:
I used the `cert-manager` project as a basis for how they generate their static deployment files from the Helm chart that has been created for the project. This value will be used to template out the Namespace API object from the Helm chart